plans/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+from . import conf as app_settings
+
+
+class PlansConfig(AppConfig):
+    name = 'plans'
+    verbose_name = _(app_settings.APP_VERBOSE_NAME)
```

Comment:
`ugettext_lazy` won't work like this.
You cannot mark variables for translations.

If someone adds a custom `APP_VERBOSE_NAME` they should mark it for translation when they define it.